exercises/bowling/example.rb

```diff
@@ -20,7 +20,7 @@ def roll(pins)
   end
 
   def validate(pins)
-    raise 'Invalid number of pins' unless (RULES[:MIN]..RULES[:MAX]).cover?(pins)
+    raise 'Invalid number of pins' unless (PINS[:MIN]..PINS[:MAX]).cover?(pins)
```

Comment:
That's a good point. I guess the xcommon data doesn't test for non-integer numbers given that it needs to support languages that are typed?